app/models/concerns/profile_tailoring.rb

```diff
@@ -2,6 +2,8 @@
 
 # Methods that are related to profile tailoring
 module ProfileTailoring
+  GROUP_ANCESTRY_IDS = Arel.sql("unnest(string_to_array(ancestry, '/'))::uuid")
```

Comment:
This is not arel, you just marked the string to bypass any static analysis toool to handle it as malicious. You should have something like [this](https://github.com/RedHatInsights/compliance-backend/blob/master/app/models/os_major_version.rb#L9-L34).